package.json

```diff
@@ -74,6 +74,8 @@
     "@rollup/plugin-json": "^6.0.0",
     "@rollup/plugin-node-resolve": "^15.0.2",
     "@rollup/plugin-typescript": "^11.1.0",
+    "@testing-library/dom": "^9.2.0",
```

Comment:
and for this one I thought we have https://github.com/containers/podman-desktop/blob/main/packages/renderer/package.json#L18